index.html

```diff
@@ -7,44 +7,38 @@
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&display=swap" rel="stylesheet"/>
-    <link rel="stylesheet" href="./stylesheets/common.css">
-    <link rel="stylesheet" type="text/css" media="screen and (max-width: 999px)" href="./stylesheets/M_mobile.css">
-    <link rel="stylesheet" type="text/css" media="screen and (max-width: 999px)" href="./stylesheets/M_cards.css">
-    <link rel="stylesheet" type="text/css" media="screen and (max-width: 999px)" href="./stylesheets/M_shower.css">
-    <link rel="stylesheet" type="text/css" media="screen and (min-width: 1000px)" href="./stylesheets/D_desktop.css">
-    <link rel="stylesheet" type="text/css" media="screen and (min-width: 1000px)" href="./stylesheets/D_cards.css">
-    <link rel="stylesheet" type="text/css" media="screen and (min-width: 1000px)" href="./stylesheets/D_shower.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
     <title>Portfolio</title>
 </head>
 <body>
 
     <header>
-        <p><a href="#" class="logo">StAnLEY</a></p>
+        <p><a href="#" >StAnLEY</a></p>
         <nav>
-            <ul class="hamburger">
-                <li class="drop"><a href="#"></a></li>
-                <li class="drop"><a href="#"></a></li>
-                <li class="drop"><a href="#"></a></li>
+            <ul >
+                <li ><a href="#"></a></li>
+                <li ><a href="#"></a></li>
+                <li ><a href="#"></a></li>
             </ul>
-            <ul class="nav-link">
+            <ul >
                 <li><a href="#">Portfolio</a></li>
                 <li><a href="#">About</a></li>
                 <li><a href="#">Contact</a></li>
             </ul>
         </nav>
     </header>
     <main>
-        <section class="hero">
-            <div class="hero-container">
+        <section >
+            <div >
             <h1>I’m Stanley.<br>Glad to see you!</h1>
-            <p class="content">
+            <p >
```

Comment:
it's not allowed to use a semantic break tag to the h1 heading. Kindly use the appropriate, the link should be helpful https://www.w3schools.com/html/html5_semantic_elements.asp